system/include/webgpu/webgpu.h

```diff
@@ -996,6 +998,7 @@ typedef struct WGPURenderPassColorAttachment {
     WGPULoadOp loadOp;
     WGPUStoreOp storeOp;
     WGPUColor clearColor;
+    WGPUColor clearValue;
```

Comment:
clearValue replaces clearColor, depthClearValue replaces clearDepth, and stencilClearValue replaces clearStencil. We don't usually do deprecation-periods in Emscripten, so they shouldn't be duplicated in this header (and struct_info). Looks like dawn.json is just missing the 'deprecated' tag on the 3 old members, to generate this correctly. https://dawn-review.googlesource.com/c/dawn/+/80941

(The naming change happened because these used to be header-only, but now also exist in the JS API with different names. We'll have to do an upstream webgpu.h update too.)